routes/get-current-project.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(config) {
 
     if(!user) {
       if(!req.session.project.remixId) {
-        sendResponse(res, project, DefaultProject.map(function(f) {
+        sendResponse(res, project, defaultProject.get("stay-calm").map(function(f) {
           var file = JSON.parse(JSON.stringify(f));
```

Comment:
Why do you ask? The way this is set up it won't be. Are you wondering why we don't use a stream instead in this case?